ConfiguringNotebookEnvironment.ipynb

```diff
@@ -4,7 +4,9 @@
    "cell_type": "markdown",
```

Comment:
<p>Looks like you still have relative paths in here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Fixed the URLs - thanks for spotting this todo that I'd forgotten about</p>